.github/renovate.json

```
@@ -51,7 +52,7 @@
     },
     {
       "matchDepTypes": ["devDependencies"],
-      "matchPackagePatterns": ["^jest-", "jest$", "^prettier", "prettier$"],
+      "matchPackagePatterns": ["^jest-", "jest$", "^prettier", "prettier$", "lint-staged"],
```

```suggestion
      "matchPackagePatterns": ["^jest-", "jest$", "lint-staged", "markdownlint-cli", "^prettier", "prettier$"],
```

---

```suggestion
      "matchPackagePatterns": ["^jest-", "jest$", "lint-staged", "lock file-lint", "markdownlint-cli", "^prettier", "prettier$"],
```

@TimDiekmann Sorry, one more... and alphabetized.

---

Addressed in https://github.com/hashintel/hash/pull/3050/commits/48b25719756dd2e50f3c83c6412871e479586bdd